### PR TITLE
update styles/classes issue in EventHistory + peerNodeCount parsing bugs

### PR DESCRIPTION
**Pull request by amazingandyyy**

### CODE REVIEW

**Comment by amazingandyyy** [comment]
fix https://github.com/bodhiproject/bodhi-ui/issues/751

File: src/scenes/Activities/scenes/EventHistory.js [hunk 1]
```diff
@@ -128,14 +131,13 @@ export default class EventHistory extends Component {
...
 
   render() {
-    const { classes } = this.props;
     const { transactions } = this.state;
 
     return (
       <Grid container spacing={0}>
         {
           transactions.length ?
-            (<Table className={classes.historyTable}>
```
Comment (henryzbw) [inline src/scenes/Activities/scenes/EventHistory.js]: don't remove style, follow the design by adding corresponding styles. You can find the styles in previous commit.

File: src/scenes/Activities/scenes/EventHistory.js [hunk 2]
```diff
@@ -307,8 +309,8 @@ export default class EventHistory extends Component {
     const isExpanded = this.state.expanded.includes(txid);
 
     result[0] = (
-      <TableRow key={txid} selected={isExpanded} onClick={this.handleClick(txid)} className={classes.clickToExpandRow} >
```
Comment (henryzbw) [inline src/scenes/Activities/scenes/EventHistory.js]: don't remove style, follow the design by adding corresponding styles. You can find the styles in previous commit.

File: src/scenes/Activities/scenes/EventHistory.js [hunk 3]
```diff
@@ -307,8 +309,8 @@ export default class EventHistory extends Component {
...
-        <TableCell className={classes.summaryRowCell}>
```
Comment (henryzbw) [inline src/scenes/Activities/scenes/EventHistory.js]: don't remove style, follow the design by adding corresponding styles. You can find the styles in previous commit.

File: src/scenes/Activities/scenes/EventHistory.js [hunk 4]
```diff
@@ -335,7 +337,7 @@ export default class EventHistory extends Component {
     );
     result[1] = (
       <TableRow key={`txaddr-${txid}`} selected onClick={this.handleClick(txid)} className={isExpanded ? classes.show : classes.hide}>
-        <TransactionHistoryAddress transaction={transaction} className={classes.detailRow} />
```
Comment (henryzbw) [inline src/scenes/Activities/scenes/EventHistory.js]: don't remove style, follow the design by adding corresponding styles. You can find the styles in previous commit.

File: src/scenes/Activities/scenes/EventHistory.js [hunk 5]
```diff
@@ -404,8 +406,8 @@ export default class EventHistory extends Component {
   }
 }
 
-const NameLinkCell = ({ classes, clickable, topic, ...props }) => (
+const NameLinkCell = ({ clickable, topic, ...props }) => (
   <TableCell>
-    <span className={clickable && classes.viewEventLink} {...props} />
```
Comment (henryzbw) [inline src/scenes/Activities/scenes/EventHistory.js]: don't remove style, follow the design by adding corresponding styles. You can find the styles in previous commit.

File: src/scenes/Activities/scenes/EventHistory.js [hunk 6]
```diff
@@ -128,14 +131,13 @@ export default class EventHistory extends Component {
   }
 
   render() {
-    const { classes } = this.props;
     const { transactions } = this.state;
 
     return (
       <Grid container spacing={0}>
         {
```
Comment (alex-cory) [inline src/scenes/Activities/scenes/EventHistory.js]: can you fix this too plz :)
```js
{transactions.length ? (

) : (

)}
```

File: src/scenes/Activities/scenes/EventHistory.js (hunk 1 above)
Comment (amazingandyyy) [reply src/scenes/Activities/scenes/EventHistory.js]: uh, I remove because I didn't find it tbh.

Comment (amazingandyyy) [reply src/scenes/Activities/scenes/EventHistory.js]: even back to https://github.com/bodhiproject/bodhi-ui/commit/bc501aefc1b25c22b5ad66a0f38fe8e2049763c3 I still didn't see the historyTable anywhere. 

if you use the page, nothing wrong to be seen now.

File: src/scenes/Activities/scenes/EventHistory.js (hunk 2 above)
Comment (amazingandyyy) [reply src/scenes/Activities/scenes/EventHistory.js]: comment above

File: src/scenes/Activities/scenes/EventHistory.js (hunk 3 above)
Comment (amazingandyyy) [reply src/scenes/Activities/scenes/EventHistory.js]: comment above

File: src/scenes/Activities/scenes/EventHistory.js (hunk 4 above)
Comment (amazingandyyy) [reply src/scenes/Activities/scenes/EventHistory.js]: comment above

File: src/scenes/Activities/scenes/EventHistory.js (hunk 5 above)
Comment (amazingandyyy) [reply src/scenes/Activities/scenes/EventHistory.js]: comment above

File: src/scenes/Activities/scenes/EventHistory.js (hunk 1 above)
Comment (amazingandyyy) [reply src/scenes/Activities/scenes/EventHistory.js]: even back to https://github.com/bodhiproject/bodhi-ui/commit/bc501aefc1b25c22b5ad66a0f38fe8e2049763c3 I still didn't see the `historyTable`, `clickToExpandRow`, `detailRow`, `viewEventLink ` anywhere. 

if you use the page, nothing wrong to be seen now.

File: src/scenes/Activities/scenes/EventHistory.js (hunk 6 above)
Comment (amazingandyyy) [reply src/scenes/Activities/scenes/EventHistory.js]: good suggestion, updated

File: src/scenes/Activities/scenes/EventHistory.js (hunk 1 above)
Comment (amazingandyyy) [reply src/scenes/Activities/scenes/EventHistory.js]: or can you point me to the original table dropdown design? I can check if those missing classess are necessary.

File: src/scenes/Activities/scenes/EventHistory.js [hunk 7]
```diff
@@ -314,7 +317,7 @@ export default class EventHistory extends Component {
         <TableCell>
           {getTxTypeString(type, locale, localeMessages)}
         </TableCell>
-        <NameLinkCell clickable={topic && topic.address} onClick={this.handleClick(txid, topic && topic.address)}>
+        <NameLinkCell clickable={topic && topic.address} onClick={this.handleClick(txid, topic && topic.address)} classes>
```
Comment (alex-cory) [inline src/scenes/Activities/scenes/EventHistory.js]: classes??

Comment (amazingandyyy) [reply src/scenes/Activities/scenes/EventHistory.js]: yes, that how we passing to 

```
const NameLinkCell = ({ classes, clickable, topic, ...props }) => (
  <TableCell>
    <span className={clickable && classes.viewEventLink} {...props} />
  </TableCell>
);
```

Comment (amazingandyyy) [reply src/scenes/Activities/scenes/EventHistory.js]: in line 407

Comment (amazingandyyy) [reply src/scenes/Activities/scenes/EventHistory.js]: but I updated, hope it looks better now

**Comment by amazingandyyy** [comment]
need to merge [#60](https://github.com/bodhiproject/bodhi-server/pull/60/files) first.

File: src/stores/GlobalStore.js [hunk 8]
```diff
@@ -18,11 +19,12 @@ export default class GlobalStore {
     if (syncInfo.error) {
       console.log(syncInfo.error.message); // eslint-disable-line no-console
     } else {
-      const { percent, blockNum, blockTime, balances } = new SyncInfo(syncInfo);
+      const { percent, blockNum, blockTime, peerNodeCount, balances } = new SyncInfo(syncInfo);
       runInAction(() => {
```
Comment (alex-cory) [inline src/stores/GlobalStore.js]: remove `runInAction`

**Comment by amazingandyyy** [comment]
@alex-cory  I want to land this this morning thanks :D

File: src/stores/models/SyncInfo.js [hunk 9]
```diff
@@ -16,10 +16,10 @@ export default class SyncInfo {
     this.percent = syncInfo.syncPercent;
     this.blockNum = syncInfo.syncBlockNum;
     this.blockTime = Number(syncInfo.syncBlockTime);
-    this.balances = _.map(syncInfo.addressBalances, (addressBalance) => new AddressBalance(addressBalance));
-    this.peerNodeCount = syncInfo.peerNodeCount;
+    this.peerCount = Number(syncInfo);
```
Comment (alex-cory) [inline src/stores/models/SyncInfo.js]: is this supposed to be `syncInfo.peerNodeCount`?

Comment (amazingandyyy) [reply src/stores/models/SyncInfo.js]: god good catch 
many thankssss

**Pull request merged**